### ISSUE TITLE
Attribute ping is missing on AnchorHTMLAttributes

### ISSUE THREAD

**Issue by janerikmai** [ionic-team/stencil#5751]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md).
- [X] I agree to follow the [Code of Conduct](https://github.com/ionic-team/stencil/blob/main/CODE_OF_CONDUCT.md).
- [X] I have searched for [existing issues](https://github.com/ionic-team/stencil/issues) that already report this problem, without success.

### Stencil Version

4.18.0

### Current Behavior

Property 'ping' does not exist on type 'AnchorHTMLAttributes<HTMLAnchorElement>'.ts(2322) https://github.com/DefinitelyTyped/DefinitelyTyped/blob/18753976aad3956a963c3cafeeab6755b7c879a3/types/react/index.d.ts#L3105

### Expected Behavior

```jsx
    render() {
        return (
            <Host>
              <a href='https://www.google.com/' ping='https://www.google.com/tracking'>Anchor with ping attribute</a>
```
should be fine

### System Info

```shell
System: node 20.11.1
    Platform: darwin (23.4.0)
   CPU Model: Apple M2 Pro (12 cpus)
    Compiler: ***/node_modules/@stencil/core/compiler/stencil.js
       Build: 1715010794
     Stencil: 4.18.0 🍵
  TypeScript: 5.4.5
      Rollup: 2.56.3
      Parse5: 7.1.2
      jQuery: 4.0.0-pre
      Terser: 5.31.0
```

### Steps to Reproduce

```jsx
import { Component, Element, Host,  h } from '@stencil/core';

@Component({
  tag: 'my-anchor',
  shadow: true
}) 
export class MyAnchor {
  @Element() host;

  render() {
    return (
      <Host>
        <a href='https://www.google.com/' ping='https://www.google.com/tracking'>Anchor with ping attribute</a>
      </Host>  
    )
  }
}
```

### Code Reproduction URL

https://www.github.com

### Additional Information

<img width="713" alt="Screenshot 2024-05-10 at 14 37 24" src="https://github.com/ionic-team/stencil/assets/2346648/fc43642d-045c-4251-a013-e83b07c68357">
<br />
should fix that...

**Comment by alicewriteswrongs** [comment]
hey @janerikmai, was just able to confirm this! I can't ping!

<img width="987" alt="Screenshot 2024-05-10 at 11 04 18 AM" src="https://github.com/ionic-team/stencil/assets/6207644/f828bba7-1f54-4e00-8f5e-0bb9cb454656">

I just labeled this so the team can take a look at your proposed fix in #5752, thanks for providing a reproduction and opening a fix!

**Comment by alicewriteswrongs** [comment]
@janerikmai hey I just put that PR to merge, it will be included in our next release -- we'll ping here on this issue when that goes out!

Thanks for your contribution 🎉

**Comment by alicewriteswrongs** [comment]
The fix for this issue has been released as a part of today's [Stencil v4.18.1 release](https://github.com/ionic-team/stencil/releases/tag/v4.18.1)!
